### PR TITLE
Fix incorrect error message in virsh_vol_download_upload

### PR DESCRIPTION
**Pull request by Yingshun**
Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
Any result?

**Comment by Yingshun** [comment]
Test result:
` (1/1) type_specific.io-github-autotest-libvirt.virsh.vol_download_upload.mix_download_upload.dir_pool.sparse: FAIL: download image size:135266304 is less than original size:139460608 or larger than 1G. (27.35 s)`

File: libvirt/tests/src/virsh_cmd/volume/virsh_vol_download_upload.py [hunk 1]
```diff
@@ -375,8 +375,10 @@ def get_pre_post_digest():
             one_g_in_bytes = 1073741824
             download_img_info = utils_misc.get_image_info(download_file_path)
             download_disk_size = int(download_img_info['dsize'])
-            if download_disk_size < original_disk_size or download_disk_size > one_g_in_bytes:
-                test.fail("download image size:%d is greater than original size:%d or larger than 1G"
+            if (download_disk_size < original_disk_size or
```
Comment (chloerh) [inline libvirt/tests/src/virsh_cmd/volume/virsh_vol_download_upload.py:378]: Isn't the brackets unnecessary?

Comment (Yingshun) [reply libvirt/tests/src/virsh_cmd/volume/virsh_vol_download_upload.py:378]: There are more than 80 characters, just split them into 2 lines. so it's necessary.

**Pull request merged**